{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly.
+
+### PR Cover Letter
+- [ ] Include A detailed description of the changes in the body of the PR
+
+### Testing
+- [ ] Unit, integration, acceptance tests as appropriate
+- [ ] Database migrations are backwards-compatible
+
+### DevOps assistance
+If it's needed, or even if you suspect it might be needed, do any/all these
+things as early as possible. Don't surprise devops right before merging!
+For example, touching envs/aws.py or or envs/common.py are dead giveaways
+that you'll need assistance when pushing your changes out.
+
+Things that require devops attention include, but are not limited to:
+- configuration changes
+- feature flags
+- migrations
+
+Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
+- [ ] Tag devops on your PR (`@edx/devops`), either for review (more specific to devops = better) or just a general FYI
+- [ ] File a support ticket
+
+FYI: Tag anyone who might be interested in this PR here.
+
+### Post-review
+- [ ] Squash commits
+
+## Reviewers
+If you've been tagged for review, please check your corresponding box once you've given the :+1:.
+- [ ] Code review #1
+- [ ] Code review #2
+- [ ] Docs review (required for UI strings/error messages)
+- [ ] UX review
+- [ ] Accessibility review
+- [ ] Product review
+
+### Areas to Consider
+- [ ] i18n 
```

Comment:
T&L has an i18n and RTL testing policy that it would be good to have everyone adopt. We require that folks upgrade the translations on their sandbox that they provide for review, and that they have themselves manually verified on this sandbox that i18n and RTL works as expected. This obviously is only relevant for PRs with new UI, but that's also the case for a11y.